test/onnx/test_op_consistency.py

```diff
@@ -336,21 +340,50 @@ def reason_flaky() -> str:
         reason=reason_onnx_does_not_support("Ceil")
     ),
     skip("ceil", dtypes=[torch.float64], reason=reason_onnx_runtime_does_not_support("Ceil", ["f64"])),
+    xfail(
+        "div", variant_name="no_rounding_mode", dtypes=COMPLEX_TYPES,
+        reason=reason_jit_tracer_error("complex types")
+    ),
+    xfail(
+        "div", variant_name="floor_rounding", dtypes=COMPLEX_TYPES,
+        reason=reason_jit_tracer_error("complex types")
+    ),
+    xfail(
+        "div", variant_name="trunc_rounding", dtypes=(torch.float16,) + COMPLEX_TYPES,
+        reason=reason_jit_tracer_error("f16 and complex types")
+    ),
+    skip(
+        "div", variant_name="no_rounding_mode", dtypes=[torch.uint8, torch.int8, torch.int16],
+        reason=reason_onnx_runtime_does_not_support("Div", ["u8", "i8", "i16"])
+    ),
+    skip(
+        "div", variant_name="floor_rounding", dtypes=[torch.uint8, torch.int8, torch.int16],
+        reason=reason_onnx_runtime_does_not_support("Div", ["u8", "i8", "i16"])
+    ),
+    skip(
+        "div", variant_name="floor_rounding", dtypes=[torch.float64],
+        reason=reason_onnx_runtime_does_not_support("Div", ["f64"])
+    ),
+    skip(
+        "div", variant_name="trunc_rounding", dtypes=[torch.uint8, torch.int8, torch.int16],
+        reason=reason_onnx_runtime_does_not_support("Div", ["u8", "i8", "i16"])
+    ),
+    xfail("floor_divide", dtypes=COMPLEX_TYPES, reason=reason_jit_tracer_error("complex types")),
+    skip("floor_divide", dtypes=[torch.float64], reason=reason_onnx_runtime_does_not_support("Floor", ["f64"])),
```

Comment:
should it be xfail, since this fails all the time, and is not flaky test? (how about rename to `skip_flaky` in the first PR of the stack, if we only skip flaky and xfail for failures)

Reply:
skip_flaky sounds great; I think this was flaky but not completely sure